test/dummy/config/boot.rb

```diff
@@ -4,7 +4,7 @@
 gemfile = File.expand_path('../../../Gemfile', __dir__)
 
 if File.exist?(gemfile)
-  ENV['BUNDLE_GEMFILE'] = gemfile
+  ENV['BUNDLE_GEMFILE'] ||= gemfile
```

Comment:
For context this configuration was the key the previous bundler issues. This configuration was previously resetting the `BUNDLE_GEMFILE` to the top level `Gemfile` once the dummy app was [loaded](https://github.com/Invoca/aggregate/blob/6cce9804f01557c85e7e457662e0af7acc825ed8/Rakefile#L26) in the raketask. In doing so it was essentially running all the appraisal steps against the version of rails in the Gemfile.lock file. Fixing this exposed 1 test failure in rails 5+ that I've addressed in the latest [commit](https://github.com/Invoca/aggregate/pull/138/commits/994358632e3aedb985400b805bcc11e084fb6a9a).

Reply:
Yea, I'm pretty sure the code you removed was added as part of the upgrade when we started pulling the gem into web and noticed this was required.